Source/WebCore/Modules/fetch/FetchLoader.h

```diff
@@ -43,6 +43,8 @@ class ScriptExecutionContext;
 class FragmentedSharedBuffer;
 
 class WEBCORE_EXPORT FetchLoader final : public ThreadableLoaderClient {
+    WTF_MAKE_FAST_ALLOCATED_WITH_HEAP_IDENTIFIER(Loader);
+    WTF_OVERRIDE_DELETE_FOR_CHECKED_PTR(FetchLoader);
```

Comment:
What happens if someone creates a subclass and forget to use these macros?
Do we get a compilation error/warning? Or will it silently fail?

Reply:
If a class hierarchy forgets WTF_OVERRIDE_DELETE_FOR_CHECKED_PTR entirely, you get a compilation error.

If a base class includes WTF_OVERRIDE_DELETE_FOR_CHECKED_PTR, but a subclass forgets it, then you get a runtime assertion when you delete the subclass object. (I tried hard to make that a compilation error, but I couldn't figure a way. Ultimately I decided that a runtime assertion is OK, since you get a deterministic failure in any debug run of the regression tests. IsoHeaps behave similarly.)